quilt/tools/command.py

```diff
@@ -721,11 +737,94 @@ def install(package, hash=None, version=None, tag=None, force=False):
 
     pkgobj.save_contents()
 
+def _setup_env(env, files):
+    """ process data distribution. """
+    # TODO: build.yml is not saved in the package system, so re-load it here
+    environments = None
+    with open('build.yml') as fd:
+        buildfile = next(yaml.load_all(fd), None)
+        environments = buildfile.get('environments')
+    if env != 'default' and (environments is None or env not in environments):
+        raise CommandException(
+            "environment %s not found in environments: section of build.yml" % env)
+    if environments is None:
+        return files
+    if env == 'default' and 'default' not in environments:
+        return files
+    print('processing environment %s: checking data...' % (env))
+    environment = environments[env]
+    dataset = environment.get('dataset')
+    for key, val in files.items():
+        # TODO: debug mode, where we can see which files were skipped
+        if type(val) == pd.core.frame.DataFrame:
```

Comment:
Use `isinstance`, to allow subclasses. Also, `pd.DataFrame` is the same thing.

Reply:
DONE